src/main/java/formflow/library/interceptors/SessionContinuityInterceptor.java

```diff
@@ -45,7 +45,7 @@ public SessionContinuityInterceptor(List<FlowConfiguration> flowConfigurations)
    */
   @Override
   public boolean preHandle(HttpServletRequest request, @NotNull HttpServletResponse response, @NotNull Object handler)
-      throws IOException {
+      throws Exception {
```

Comment:
I think this might be clearer if it's `IOException, LandmarkNotSetException` to make it clear to folks exactly what they have to handle. 

Also the javadocs above need to include the two exceptions.

Reply:
To share more of my thinking. It's good practice to be specific when you can, hence using the specific exceptions rather than just saying it `throws Exception`. Technically that's true, but not very helpful to developers if they are quickly reading through the code.